src/twisted/newsfragments/7530.bugfix

```diff
@@ -0,0 +1 @@
+Allow for key and cert .pem files that do not have a a trailing newline
```

Comment:
Unless we change crypto/openssl to handle the two keys with no newline between or parse it apart ourself right before passing it to crypto, it seems we do need the fix where it is.  But, I'll agree that may well not fix all cases.  I'm digging a bit to see if i can find, test, and fix the others.